### DISCUSSION TITLE
watchOS app?

### DISCUSSION THREAD

**Discussion by SkipsterTheHipster** [advplyr/audiobookshelf-app#530] [General]
Are there any plans to design a watchOS companion app? If not, would you have any problem with me using the code to work on creating one? I've never built an app, let alone a watchOS app, and since I would really like to have an Audiobookshelf app on my Apple Watch, I thought I might look at this as my first app development experience.

What do you say, @advplyr?

**Comment by advplyr** [comment]
I don't have an Apple Watch and never built anything for watchOS so I haven't thought about it.
I would encourage others to build clients for Abs. I know a few have already built their own mobile/desktop interfaces.

You shouldn't need to use any code from the existing repos for this. There is a documented API https://api.audiobookshelf.org
